charts/incubator/dendrite/values.yaml

```diff
@@ -115,6 +132,8 @@ appserviceapi:
         internal:
           enabled: true
           port: 7777
+  ingress:
```

Comment:
that `ingress` is duplicated -> see line 39

Reply:
Will remove when I add the well-known paths. Currently dendrite only supports the server one, I have raised a PR (matrix-org/dendrite#2551) to add the client route, so I will hold off on opening another PR for the chart until that is merged and released.